Composer/packages/client/src/pages/publish/publishPageUtils.tsx

```diff
@@ -28,14 +31,26 @@ export const generateBotPropertyData = (botProjectData: BotProjectType[]) => {
   return { botPropertyData, botList };
 };
 
+const findSkillManifestUrl = (skillManifests: SkillManifestFile[], appId: string) => {
+  for (const skillManifest of skillManifests || []) {
```

Comment:
```.find()```

Reply:
I don't want to find the skill manifest, I want to find the endpoint.